xamarin/Xamarin.Forms
# [Android] Fix ScrollView visibility issue

### Description of Change ###

In this PR https://github.com/xamarin/Xamarin.Forms/pull/8090 was added the support to be able to manage the visibility of the scrollbar in the ScrollView on Android. However, this PR added a repeated issue, the scrollbar visibility the child elements of the ScrollView when initializing (or refreshing). For this reason, this PR https://github.com/xamarin/Xamarin.Forms/pull/10447 revert original one changes.

This PR recovers the initial changes to allow managing the visibility of the scrollbar in ScrollView but also avoiding the issue with the child elements.

![fixed-scrollview-visibility-droid](https://user-images.githubusercontent.com/6755973/81686380-c9174600-9458-11ea-9ae2-e0614e2ecfaa.gif)

The problem was in the used Context. When creating `ScrollViewRenderer` we were creating a context that used a custom style (using vertical and horizontal scrollbars). This context was used creating `ScrollViewContainer`. So by default `ScrollViewContainer` had `VerticalScrollBarEnabled` set to true (incorrect!).

### Issues Resolved ### 

- fixes #8015
- fixes #9103

### API Changes ###
 
 None

### Platforms Affected ### 

- Android

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

#### Before
![issue-scrollbar-scrollview](https://user-images.githubusercontent.com/6755973/81686396-cae10980-9458-11ea-9d6d-a43119d1b1f6.gif)
#### After
![fixed-scroll-droid](https://user-images.githubusercontent.com/6755973/81686359-c61c5580-9458-11ea-8dbf-b1a457f0e871.gif)

### Testing Procedure ###
Launch Core Gallery. Navigate to the "ScrollView Gallery" to verify that can hide/show the ScrollView toolbar. Also, navigate to the "Material Entry Gallery" (for example) and set the focus on different Entries to validate that the "flickering scrollbar" issue in the ScrollView children's is fixed.

### PR Checklist ###
